assets/app/view/user.rb

```diff
@@ -89,9 +91,17 @@ def reset_settings
       input_elm(:bg2).value = default_for(:bg2)
       input_elm(:font2).value = default_for(:font2)
       input_elm(:red_logo).checked = false
+
       TILE_COLORS.each do |color, hex_color|
         input_elm(color).value = hex_color
       end
+
+      ROUTE_COLORS.each_with_index do |hex_color, index|
+        input_elm("r#{index}_color").value = hex_color
```

Comment:
use your function that you added here?

Reply:
I’m not reading sth. there. This is just resetting the input-value.

Reply:
i don't want "r#{index_x}" everywhere in the code can we clean that up

Reply:
The inputs need to have a unique id. This is the easiest solution to generate one known to me and easy to understand, too.

Reply:
can you create a helper function that generates this key?